app-feature-preview/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_logo"/>
+    <monochrome android:drawable="@drawable/ic_launcher_logo" />
```

Comment:
`@drawable/ic_launcher_logo` is not a monochrome icon.

Reply:
See above